src/virtual_ptdf_calculations.jl

```diff
@@ -174,50 +284,40 @@ function Base.isempty(vptdf::VirtualPTDF)
     !isempty(vptdf.temp_data) && return false
     return true
 end
-# Size related overload will work on the BA matrix
+
+"""
+Gives the size of the whole PTDF matrix, not the number of rows stored.
```

Comment:
Check the wording for `Base.size` it reads 

```suggestion
Return a tuple containing the dimensions of the VirtualPTDF, not the number of rows stored.
```